Task1.1/task1.js

```diff
@@ -0,0 +1,16 @@
+const through2 = require("through2");
```

Comment:
Please use typescript instead of js.

Reply:
Task 1 was in commeonJS module so didn't use it